src/app/app.component.spec.ts

```diff
@@ -7,7 +7,7 @@ import { Router } from '@angular/router';
 import { MockComponent } from 'ng2-mock-component';
 import { NgReduxTestingModule, MockNgRedux } from '@angular-redux/store/testing';
 import { AuthService } from './services/auth/auth.service';
-import { MenuModule } from './components/menu/menu.module';
+import { CapDashModule } from 'capdash2-common-module/src/lib/module';
 import { ItemModule } from './core/ajax/item/item.module';
```

Comment:
This is ok for now but we may need to use a different naming convention since this is meant to be used for more than just CapDash.